src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -120,13 +120,35 @@ message WorkerInitRequest {
 
 // Worker responds with the result of initializing itself
 message WorkerInitResponse {
-  // Version of worker
+  // NOT USED
+  // TODO: Remove from protobuf during next breaking change release
   string worker_version = 1;
```

Comment:
For understanding, these comments indicate this property not being used vs. the entire message, right?

Reply:
Yup, just referring to the property!

Reply:
Okay! I read it at first as the message since it's right underneath, could maybe be worth specifying "property not used" or something like that for clarity? But if these changes are already in the main protobuf repo makes sense to not have them out of sync.

Reply:
I figured putting the comment above the property and not above the message (like with [WorkerHeartbeat](https://github.com/Azure/azure-functions-language-worker-protobuf/blob/76bfa387315eba37ddf96f1805e60d18cde17bdc/src/proto/FunctionRpc.proto#L176-L178)) would be clear enough but I can see the confusion here - we can update the comment to "property not used" so it's more clear. Sadly will have to do this in the protobuf repo with the next release

Reply:
PR for this here https://github.com/Azure/azure-functions-language-worker-protobuf/pull/80